scripts/run_self_reg.sh

```diff
@@ -2,5 +2,10 @@
 
 echo Start mod-licenses in self-register mode
 
-java -jar build/libs/mod-license-1.0.jar -Xmx1G --grails.server.host=10.0.2.2 --selfRegister=on --dataSource.username=folio_admin --dataSource.password=folio_admin --dataSource.url=jdbc:postgresql://localhost:5432/okapi_modules
+java -jar build/libs/mod-licenses-1.0.jar -Xmx1G \
+--grails.server.host=10.0.2.2 \
+--selfRegister=on \
+--dataSource .username=folio_admin \
+--dataSource .password=folio_admin \
+--dataSource .url=jdbc:postgresql://localhost:5432/okapi_modules
```

Comment:
Is the space after --dataSource before the dot intended?

Reply:
no, wasn't. Will change that. Somehow the line break screwed up the script and i then manually tried to fix it :(